health/health.d/vcsa.conf

```
@@ -133,8 +133,7 @@ component: VMware vCenter
    lookup: max -10s unaligned of software_packages
     units: status
     every: 10s
-     warn: $this == 4
-     crit: $this == 3
+     warn: ($this == 3) || ($this == 4)
```

```suggestion
     warn: ($this == 3) OR ($this == 4)
```

Not sure `||` is supported cc @MrZammler

---

I think I saw it in a different examples, it should work. But I would be glad to hear some opinion @MrZammler

---

Hi, yes, `||` is supported.